### ISSUE TITLE
Heroku Example is down

### ISSUE THREAD

**Issue by danielrbuchanan** [BrightcoveOS/evaporate-brightcove#28]
The example on Heroku seems to be down

**Comment by brycefisher** [comment]
Yes! So sorry. We had tons of _inappropriate_ mature content being uploaded randomly and the example was taken down :-(

**Comment by brycefisher** [comment]
Reach out to your Brightcove account manager if you're interested in giving this a test drive.

**Comment by brycefisher** [comment]
I would take it down, but I'm no longer a maintainer on this project.
